actions/workflows/st2_finalize_release.yaml

```diff
@@ -124,8 +124,30 @@ st2cd.st2_finalize_release:
                 branch: master
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - get_utc_date
+
+        get_utc_date:
+            action: core.local
+            input:
+                cmd: "date -u +%Y%m%d"
+            publish:
+                utc_date: <% task(get_utc_date).result.stdout %>
+            on-success:
+                - update_packer_st2_changelog_and_tag
+        update_packer_st2_changelog_and_tag:
```

Comment:
Let's follow already established naming following the `mistral_tag_release`

eg `vagrant_tag_release` to make it more clear.